qxf2/qxf2-survey
# Created the Landing Page for the Survey Application

## Description

Created the Landing Page for the Survey Application. I have designed and developed a React Application for the basic Frontend which is fundamentally similar to this: http://34.205.45.167/survey

## Issue Reference

The Jira Ticket is available [here](https://qxf2services.atlassian.net/jira/software/projects/QELO/boards/32?selectedIssue=QELO-45)

## Testing

- Install NodeJS and NPM on your Local Machine.

- cd into the specific directory and run the command: npm i.

- This will install all the Packages in the node_modules directory.

- Run the command: npm start to kickstart the development server.

- You can see the Application on localhost:3000.
